for_developers/program_of_work/index.md

```diff
@@ -12,7 +12,7 @@ and documented for the relevant years in the following documents:
 
 ## 2023
 
-  - [Presentation at the PH-SFT group meeting- 6th February 2022](https://indico.cern.ch/event/1233591/contributions/5189935/attachments/2588023/4465399/ROOT-2023-SFT.pdf){:target="_blank"}
+  - [Presentation at the PH-SFT group meeting- 6th February 2023](https://indico.cern.ch/event/1233591/contributions/5189935/attachments/2588023/4465399/ROOT-2023-SFT.pdf){:target="_blank"}
```

Comment:
```suggestion
  - [Presentation at the EP-SFT group meeting- 6th February 2023](https://indico.cern.ch/event/1233591/contributions/5189935/attachments/2588023/4465399/ROOT-2023-SFT.pdf){:target="_blank"}
```

Reply:
@eguiraud what about all the other occurrences of PH-SFT in the same document below?